cli/cmd/helmInstall.go

```diff
@@ -119,11 +132,13 @@ func installStack(file, ref, dbURI string) error {
 	return nil
 }
 
-func appendDBURIValues(dbURI, name string, cmds []string) ([]string, error) {
-	cmds = append(cmds, "--set",
-		"timescaledb-single.enabled=false,"+
-			"timescaledbExternal.enabled=true,"+
-			"timescaledbExternal.db_uri="+dbURI+
-			",promscale.connection.uri.secretTemplate="+name+"-timescaledb-uri,")
-	return cmds, nil
+func appendDBURIValues(dbURI, name string, helmValues string) string {
+	helmValues = helmValues+",timescaledb-single.enabled=false,"+"timescaledbExternal.enabled=true,"+"timescaledbExternal.db_uri="+dbURI+
```

Comment:
tl;dr

Reply:
This refactoring of existing logic. When connecting to external DB. Tobs needs to disabled TimescaleDB installation and map values to timescaleDBexternal fields accordingly.